code/components/extra-natives-five/src/NuiTxdSource.cpp

```diff
@@ -312,11 +316,28 @@ static InitFunction initFunction([]()
 	}, -500);
 });
 
+__int64(__fastcall* g_origAlterPedHeadshotTransparentPSConstants_Hook)(__m128*, __m128*);
+
+// Since b2189's shader changes `RegisterPedheadshotTransparent`'s texture colors are in the [0 .. 0.6] range, let's inverse it
+// Sets	PS constant buffer 3rd and 4th? (unchecked) float4/vec4/__m128 values, where the 3rd is referred to as `GeneralParams0`
+__int64 __fastcall AlterPedHeadshotTransparentPSConstants_Hook(__m128* a3, __m128* a4)
```

Comment:
__fastcall is a little redundant as that's the default on amd64 abi

Reply:
I figured I follow suit with the other examples in the repo, want it removed?

Reply:
It could stay in order to mark it as 'yeah this definition is from IDA' as the `__int64` looks equally weird like that.